packages/core/src/makePrepareRealmGlobalFromConfig.js

```diff
@@ -14,23 +14,31 @@ function makePrepareRealmGlobalFromConfig({ createFunctionWrapper }) {
     getTopLevelWriteAccessFromPackageConfig,
   }
 
+  /**
+   * Should never return an array containing `'Function'`
```

Comment:
What's this about?

Reply:
We don't allow write access to `Function` and read access is implied.

Reply:
We don't allow write access to `Function` and read access is implied.